extension-manifest-v2/src/background.js

```diff
@@ -133,6 +133,7 @@ async function tryOpenOnboarding({ force = false, period = ONE_DAY_MSEC } = {})
 	const now = Date.now();
 	if (!conf.setup_timestamp || ((now - conf.setup_timestamp) > period)) {
 		conf.setup_timestamp = now;
+		conf.setup_shown += 1;
```

Comment:
I would update that value after the tab is created/focused, not before that action.